aiohttp/web_reqrep.py

```diff
@@ -363,6 +372,8 @@ def read(self):
             while True:
                 chunk = yield from self._payload.readany()
                 body.extend(chunk)
+                if len(body) >= self._client_max_size:
+                    raise HttpRequestEntityTooLarge()
```

Comment:
For this particular case `from .web_exceptions import HTTPRequestEntityTooLarge` is allowed evil.
Yes, I know the import from function has ugly adore.
But I'm ok with this for the case.